sdk/quantum/Azure.Quantum.Jobs/README.md

```diff
@@ -113,23 +132,25 @@ var createJobDetails = new JobDetails(containerUri, inputDataFormat, providerId,
 JobDetails createdJob = (quantumJobClient.CreateJob(jobId, createJobDetails)).Value;
 ```
 
+### Get Job
+
+To retrieve a specific job by its job id, you can use this function.
+
 ```C# Snippet:Azure_Quantum_Jobs_GetJob
 // Get the job that we've just created based on its jobId
 JobDetails myJob = (quantumJobClient.GetJob(jobId)).Value;
 ```
 
-```C# Snippet:Azure_Quantum_Jobs_GetJobs
-// Get all jobs from the workspace (.ToList() will force all pages to be fetched)
-var allJobs = quantumJobClient.GetJobs().ToList();
-```
-
-## Troubleshooting
+### Get Jobs
 
-Describe common errors and exceptions, how to "unpack" them if necessary, and include guidance for graceful handling and recovery.
+To enumerate all the jobs in the workspace, use the GetJobs() function.
 
-Provide information to help developers avoid throttling or other service-enforced errors they might encounter. For example, provide guidance and examples for using retry or connection policies in the API.
-
-If the package or a related package supports it, include tips for logging or enabling instrumentation to help them debug their code.
+```C# Snippet:Azure_Quantum_Jobs_GetJobs
+foreach (JobDetails job in quantumJobClient.GetJobs())
+{
+    Console.WriteLine($"{job.Name}");
+}
+```
 
 ## Next steps
```

Comment:
Delete everything that isn't an actual next step.  Maybe say something like "Visit our documentation at http://xyz.com to learn more about Azure Quantum."?